dev-lang/rust/rust-1.21.0.recipe

```diff
@@ -0,0 +1,211 @@
+CHECKSUM_SHA256_5="e40ca6485049911c319b717404181675e88c2d56d2d8d27395cc85b0e4f428fb"
+CHECKSUM_SHA256_6="e5481733ae30cc985674b7494e48ba8717748af9265514dd8516ceceb32ae8f0"
+CHECKSUM_SHA256_7="6740e2f8d035a5a5246a12e84a141d37ad4c2b826355aff497e6f1fcfb637aef"
+CHECKSUM_SHA256_8="d3e255dfe63cabd342229a53605cd94ae6df912e4771f88a1a4de7af3a59d8f2"
+CHECKSUM_SHA256_9="7e0fb24af053ac9415568b01442a5ab491a9ed5b841e9441808100d63175cb68"
+CHECKSUM_SHA256_10="e2059881a339efff63e94d363b75bf34dfad2a0297bc46c568ba40dcc71f2d6f"
+CHECKSUM_SHA256_11="fa74c46f239ebd1557dea184de3caf7d76c003c6de64aa7fc07c2c9d9f0543f3"
+CHECKSUM_SHA256_12="82fa7ada1bdadbba9f780847dc4f9340f15e5f137d838c98ca7e30ec3dda6684"
+SOURCE_DIR="rust-$srcGitRev"
+PATCHES="rust-$portVersion.patchset"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	rust$secondaryArchSuffix = $portVersion
+	cmd:rustc = $portVersion
+	cmd:rustdoc = $portVersion
+	cmd:rust_gdb = $portVersion
+	cmd:rust_lldb = $portVersion
+	cmd:cargo$secondaryArchSuffix = $cargoVersion
+	cmd:rls = $rlsVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcurl$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:cmp
+	cmd:file
+	cmd:find
+	cmd:git
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python
+	cmd:sed
+	cmd:tar
+	cmd:which
+	cmd:xargs
+	rust_bin$secondaryArchSuffix = $portVersion
+	"
+CONFLICTS="
+	rust_bin$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	# With the rustbuild system, Rust always makes a tarball with the source
+	# In older recipes we symlinked the other git submodules, now we have to
+	# copy them
+	rm -rf src/tools/cargo src/libcompiler_builtins src/llvm \
+	       src/tools/rust-installer src/liblibc src/rt/hoedown \
+	       src/doc/nomicon src/doc/reference src/doc/book \
+	       src/tools/rls
+	cp -r $sourceDir3/rust-installer-$srcGitRev3 src/tools/rust-installer
+	cp -r $sourceDir4/hoedown-$srcGitRev4 src/rt/hoedown
+	cp -r $sourceDir5/libc-$srcGitRev5 src/liblibc
+	cp -r $sourceDir6/llvm-$srcGitRev6 src/llvm
+	cp -r $sourceDir7/cargo-$cargoVersion src/tools/cargo
+	cp -r $sourceDir8/nomicon-$srcGitRev8 src/doc/nomicon
+	cp -r $sourceDir9/reference-$srcGitRev9 src/doc/reference
+	cp -r $sourceDir10/book-$srcGitRev10 src/doc/book
+	cp -r $sourceDir11/rls-$srcGitRev11 src/tools/rls
+	cp -r $sourceDir12/compiler-builtins-$srcGitRev12 src/libcompiler_builtins
+	rm -rf src/libcompiler_builtins/compiler-rt
+	cp -r $sourceDir2/compiler-rt-$srcGitRev2 src/libcompiler_builtins/compiler-rt
+
+	# write the build configuration
+	cat >config.toml <<- EOL
```

Comment:
Suggestion:
```diff
-	cat >config.toml <<- EOL
+	tr -d '\t' >config.toml <<- EOL
```
(A variant would be to keep the `cat` command and remove each tab between lines 118 to 140.)